react/ProductDescription.tsx

```diff
@@ -62,19 +64,21 @@ function ProductDescription(props: PropsWithChildren<Props>) {
     return null
   }
 
-  const { collapseContent = true, title } = props
+  const { collapseContent = true, showTitle = true, title } = props
 
   return (
     <div className={handles.productDescriptionContainer}>
-      <FormattedMessage id="store/product-description.title">
-        {txt => (
-          <h2
-            className={`${handles.productDescriptionTitle} t-heading-5 mb5 mt0`}
-          >
-            {title ? formatIOMessage({ id: title, intl }) : txt}
-          </h2>
-        )}
-      </FormattedMessage>
+      {showTitle ? (
+        <FormattedMessage id="store/product-description.title">
+          {txt => (
+            <h2
+              className={`${handles.productDescriptionTitle} t-heading-5 mb5 mt0`}
+            >
+              {title ? formatIOMessage({ id: title, intl }) : txt}
+            </h2>
+          )}
+        </FormattedMessage>
+      ) : null}
```

Comment:
What do you think about this snippet? This seems to be easier to read if showTitle, then render that component.

```suggestion
      {showTitle && (
        <FormattedMessage id="store/product-description.title">
          {txt => (
            <h2
              className={`${handles.productDescriptionTitle} t-heading-5 mb5 mt0`}
            >
              {title ? formatIOMessage({ id: title, intl }) : txt}
            </h2>
          )}
        </FormattedMessage>
      )}
```

Reply:
Yes, I agree with you. It is easier to read and understand.